erigon-lib/state/aggregator.go

```diff
@@ -1646,6 +1648,7 @@ func (a *Aggregator) BeginFilesRo() *AggregatorRoTx {
 		a:       a,
 		id:      a.ctxAutoIncrement.Add(1),
 		_leakID: a.leakDetector.Add(),
+		iis:     make(map[kv.InvertedIdx]*InvertedIndexRoTx),
```

Comment:
- probably can set capacity here

Reply:
Pls rename “ ctxAutoIncrement” in another PR: to something like aggRoTxAutoIncrement

Reply:
Pls create github issue about benchmarking of agg.BeginRoFiles.